src/api2/routes/browserSetupSecurity/getBrowserSetupPreference.ts

```diff
@@ -0,0 +1,69 @@
+import { db } from '../../../helpers/db';
+import { logError, logInfo } from '../../../helpers/logger';
+import { inputSanitizer } from '../../../helpers/sanitizer';
+import { getGroupIds } from '../../helpers/bankUUID';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+export const getBrowserSetupPreference = async (req: any, res: any) => {
```

Comment:
this method is public?

Reply:
no authentication?

Reply:
yes, because It needs to be called from the vault list page when the user is not connected. I could have added a full  device authorization but a hacker would have to send a valid random device UUID to query it, so I believe that's enough for this kind of information.